website/content/en/docs/kubebuilder/operator-scope.md

```diff
@@ -0,0 +1,268 @@
+At the same time, namespace-scoped operators are preferred because of their flexibility. They enable decoupled upgrades, namespace isolation for failures and monitoring, and differing API definitions.
+
+By default, `operator-sdk init` scaffolds a cluster-scoped operator. This document details conversion of default 
+operator projects to namespaced-scoped operators. However, there are use cases where a cluster-scoped operator may make sense. For example, the [cert-manager][cert-manager] operator is often deployed with cluster-scoped permissions and watches so that it can manage and issue certificates for an entire cluster.
+
+**IMPORTANT**: When a [Manager][ctrl-manager] instance is created in the `main.go` file, the 
+Namespace(s) is set via [Manager Options][ctrl-options] as described below. These Namespace(s) should be watched and cached for the Client which is
+provided by the Controllers. Only clients provided by cluster-scoped projects are able to manage cluster-scoped CRD's. For further information see: [CRD scope doc][crd-scope-doc].
+
+## Watching resources in all Namespaces (default)
+
+A [Manager][ctrl-manager] is initialized with no Namespace option specified, or `Namespace: ""` will watch all Namespaces:
+
+```go
+...
+mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
+    Scheme:             scheme,
+    MetricsBindAddress: metricsAddr,
+    Port:               9443,
+    LeaderElection:     enableLeaderElection, 
+    LeaderElectionID:   "f1c5ece8.example.com",
+})
+...
+```
+
+**NOTE** The above example is the default code generated with `operator-sdk init` which initializes the Manager to be cluster-scoped.
+
+## Watching resources in a single Namespace
+
+To restrict the scope of the [Manager's][ctrl-manager] cache to a specific Namespace set the Namespace attribute from [Options][ctrl-options]:
+
+```go
+...
+mgr, err = ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
+    Scheme:             scheme,
+    Namespace:          "operator-namespace",
+    MetricsBindAddress: metricsAddr,
+})
+...
+``` 
+
+## Watching resources in a set of Namespaces
+
+It is possible to use the [MultiNamespacedCacheBuilder][multi-namespaced-cache-builder] attribute from 
+[Options][ctrl-options] to watch and manage resources only in a specific set of Namespaces:
+
+```go
+...
+namespaces := []string{"foo", "bar"} // List of Namespaces
+
+mgr, err = ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
+    Scheme:             scheme,
+    NewCache:           cache.MultiNamespacedCacheBuilder(namespaces),
+    MetricsBindAddress: fmt.Sprintf("%s:%d", metricsHost, metricsPort),
+})
+...
+```
+
+In the above example, if a CR(CustomResource), which is managed by its controllers, is applied to a Namespace that is not in this List of Namespaces then, its controller will not re-trigger its reconcile since the Namespaced is NOT part of the [Manager][ctrl-manager] scope.
```

Comment:
```suggestion
In the above example, a CR created in a Namespace not in the set passed to `Options` will not be reconciled by its controller because the Manager does not manage that Namespace.
```